supreme_court_predictions/statistics/datacleaner.py

```diff
@@ -90,7 +91,7 @@ def speakers_to_df(speakers_dict):
         dict_list = []
         for speaker_key in list(speakers_dict.keys()):
             speaker_data = speakers_dict[speaker_key]["meta"]
-            speaker_data["speaker_key"] = speaker_key
+            speaker_data["speaker_key"] = re.sub(r"^j__", "", speaker_key)
```

Comment:
What is the reason we don't want the `justice` part of the name? Is it because it adds a lot of text or it doesn't correlate with the other data sets?